src/freenas/usr/bin/install-dev-tools

```diff
@@ -27,7 +27,7 @@ PACKAGES=(
 )
 PIP_PACKAGES=()
 
-if [ -f /usr/local/libexec/disable-rootfs-protection ]; then
+if [ -f /usr/local/libexec/disable-rootfs-protection && -f /var/run/middleware/middlewared.sock ]; then
```

Comment:
It’s just `/run`. And what does “running on SCALE” comment below mean, where else can this be running?

Reply:
It can be running in a docker container and using the path we have defined in middleware client
```
MIDDLEWARE_RUN_DIR = '/var/run/middleware'
```
It would work well with both `/run` and `/var/run` :)